Vokoder.podspec

```diff
@@ -56,7 +56,9 @@ Pod::Spec.new do |s|
   s.subspec 'Swift' do |sw|
     sw.ios.deployment_target = '8.0'
     sw.tvos.deployment_target = '9.0'
-    sw.dependency 'Vokoder/DataSources'
+    sw.watchos.deployment_target = '3.0'
+    
+    sw.dependency 'Vokoder/Core'
```

Comment:
@chillpop When I do this in a certain application:

<img width="826" alt="screen shot 2017-03-18 at 9 54 29 am" src="https://cloud.githubusercontent.com/assets/1976498/24073206/df01fea6-0bc1-11e7-8bcc-9ce230182027.png">

It seems to work fine: 

<img width="312" alt="screen shot 2017-03-18 at 9 54 39 am" src="https://cloud.githubusercontent.com/assets/1976498/24073200/c46a15e2-0bc1-11e7-8e83-00b073f7f4bd.png">